Cargo.toml

```diff
@@ -22,6 +22,10 @@ serde = { version = "1.0", optional = true, features = ["derive"] }
 default = ["use_std"]
 use_std = []
 
+[dev-dependencies]
+serde_json = "=1.0.0"
+serde = "=1.0.0"
```

Comment:
I'd rather not have these pinned down, but I understand you did that to get them to build on 1.12, since dev-dependencies aren't optional. I'm going to see if I can work around downgrades in the CI script, as I've done other crates...